plugins/plugin-site/src/components/ThankAContributorNote.jsx

```diff
@@ -4,14 +4,9 @@ import React, {useEffect, useState} from 'react';
 import axios from 'axios';
 import dayjs from 'dayjs';
 import Papa from 'papaparse';
-import {useMediaQuery} from '../hooks/useMediaQuery';
-import './ThankAContributorNote.css';
 import placeholderImage from '../images/person-circle-outline.svg';
 
 function ThankAContributorNote() {
-    const isMobile = useMediaQuery('(max-width: 768px)');
-    const isDesktop = useMediaQuery('(min-width: 1024px)');
-
```

Comment:
This would get rid of the custom responsiveness behavior. Have you checked if it would still work without this?

Reply:
Please revert these changes, I have checked it no longer works as expected after you removed these. Also, please test carefully before pushing your code to GitHub next ime. It appears you had not tested these changes previously.

Reply:
If we want different size of the image for mobile/desktop devices, it should be defined in jenkins.io CSS, there should be no need to set these in `jsx`.

Reply:
Yes, agreed. But that is not currently done.